deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -90,6 +90,12 @@ spec:
                 types, but required for libvirt VMs driven by vbmc.
               pattern: '[0-9a-fA-F]{2}(:[0-9a-fA-F]{2}){5}'
               type: string
+            bootMode:
+              description: Select the boot mode of the system. Allowed values are UEFI,
```

Comment:
The description of the fix indicates that the default is UEFI, if so could you please add it to the description here? Also, could you please point me to the code where we default to UEFI when this value is not provided?

Reply:
@yrobla I don't see where this was resolved? There's not default value that I can see for bootmode

Reply:
see code in pkg/provisioner/ironic/ironic.go  ... it just sets it to bios if legacy is specified. In all the other cases, it is set to uefi